### PR TITLE
Disable DES3 compiler warning fix

### PR DESCRIPTION
**Pull request by ejohnstown**
Fixed unused variable complaint when KEYGEN and OPENSSL_EXTRA are enabled and when AES and DES3 are disabled.

    $ ./configure --enable-keygen --enable-opensslextra --disable-aes --disable-aesgcm --disable-des3

### CODE REVIEW

File: src/ssl.c [hunk 1]
```diff
@@ -3438,26 +3441,29 @@ static int wolfssl_encrypt_buffer_key(byte* der, word32 derSz, byte* password,
         return SSL_FATAL_ERROR;
     }
 #else
-    (void) passwordSz;
+
```
Comment (JacobBarthelmeh) [inline src/ssl.c]: Not sure that this else case is needed any longer with moving (void)passwordSz higher up.

**Comment by JacobBarthelmeh** [comment]
Can you also fix the addition of disable-md5 with this configure, it has unsued key variable in the same functions. If removing the else case with no MD5 please also remove the wolfssl_decrypt_buffer_key to match.

File: src/ssl.c (hunk 1 above)
Comment (ejohnstown) [inline src/ssl.c]: I will remove the redundant "else", but I'm treating the disable-md5 as a separate issue.

**Comment by ejohnstown** [comment]
I fixed the MD5 issue you found with

    $ ./configure --enable-keygen --enable-opensslextra --disable-aes --disable-aesgcm --disable-des3 --disable-md5

and an additional static variable that wasn't being used by these functions when AES is disabled.

**Pull request merged**